### PR TITLE
Bugfix/backend configuration: Refactor backend components with Spring Boot.

### PR DESCRIPTION
**Pull request by matbmoser**
# Why we create this PR?
 
We create this pull request to fix a important bug found in the version 0.7.0 (https://github.com/eclipse-tractusx/digital-product-pass/issues/28), this bug is related with the configuration of our backend and needed to be fixed.
 
# What we want to achieve with this PR?
 
By adding the components that are not included in spring boot, we want to fix the bug in the configuration.
With this PR we want to optimize the backend configuration and non spring boot components structure. 

After the PR is merge we need to create the release version v0.8.0 with the content of the change log.
 
# What is new?

## Updated
- Updated configuration charts
- Updated backend utilities imports (added spring boot to the classes)
- Updated tests to use spring boot to autowire the components.

## Added
- Added new configuration classes
- Added new methods to get the configurations at the start of the services

## Deleted
- Deleted the configuration util (old alternative to spring boot configuration)
- Deleted the tests for configuration util.

### CODE REVIEW

**Comment by matbmoser** [comment]
Following Checks and Chores will be performed:

- [ ] Run Dash Licenser
- [ ] Check for Missing License Headers
- [ ] Update changlog to version v0.8.0

**Comment by matbmoser** [comment]
# Notes: 
- The reason that there is **46** files changes, is related to the "utilities" being added to spring boot.
    - The "utilities" are used all over the application in the backend (so they have many instances in different files being called"

- A **end to end** test has been performed and the retrieval of the passport (the only functionality) is working.
    -  image

**Comment by matbmoser** [comment]
> Following Checks and Chores will be performed:
> 
> * [x]  Run Dash Licenser
> * [x]  Check for Missing License Headers
> * [x]  Update changlog to version v0.8.0

No IP license check is needed (all the dependencies are approved)
No Missing License Headers were found.

File: consumer-backend/productpass/src/main/java/org/eclipse/tractusx/productpass/config/PassportConfig.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/*********************************************************************************
+ *
+ * Catena-X - Product Passport Consumer Backend
```
Comment (Siegfriedk) [inline consumer-backend/productpass/src/main/java/org/eclipse/tractusx/productpass/config/PassportConfig.java:3]: Not sure if this shouldn't be Tractus-X.

Comment (matbmoser) [reply consumer-backend/productpass/src/main/java/org/eclipse/tractusx/productpass/config/PassportConfig.java:3]: We had our license headers reviewed by Angelika already and it matches with our current license file. Maybe we can schedule a meeting to revise that again. Thanks for the comment :)

**Pull request merged**